src/index.html

```diff
@@ -2,16 +2,107 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+    <meta name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="POST"
+      autocomplete="off"
+      onsubmit="onSubmit()">
+
+      <fieldset class="fieldset">
+        <legend>Personal information</legend>
+        <label for="surname" class="formField">Surname:</label>
+        <input type="text" name="surname"
+          id="surname"
+          required>
```

Comment:
```suggestion
        <input 
          type="text" 
          name="surname"
          id="surname"
          required
        >
```
якщо 3 або більше атрибутів, переноси кожен атрибут на новий рядок і знак `>`теж на новий рядок
тут і у всіх інших місцях теж